### ISSUE TITLE
node's usageNanoCores missing from kubelet stats summary for Windows nodes

### ISSUE THREAD

**Issue by zhiweiv** [kubernetes/kubernetes#75934]
**What happened**:
I am trying to make metrics-server 0.3.1 work on Windows nodes, cpu usage is missing due to the error `unable to get CPU for node "3228k8s00000000", discarding data: missing cpu usage metric` in metrics-server logs. 

In https://github.com/kubernetes-incubator/metrics-server/blob/master/pkg/sources/summary/summary.go, it throws error for nodes which UsageNanoCores  is nil.
```
if err := decodeCPU(&target.CpuUsage, nodeStats.CPU); err != nil {
	errs = append(errs, fmt.Errorf("unable to get CPU for node %q, discarding data: %v", src.node.ConnectAddress, err))
}

func decodeCPU(target *resource.Quantity, cpuStats *stats.CPUStats) error {
	if cpuStats == nil || cpuStats.UsageNanoCores == nil {
		return fmt.Errorf("missing cpu usage metric")
	}

	*target = *uint64Quantity(*cpuStats.UsageNanoCores, -9)
	return nil
}
```
After turned on metrics-server verbose logs, I can see UsageNanoCores is really missing from node for Windows, it exists in Windows pods and containers, this is from metrics-server verbose logs, I removed irrelevant part.
```
{
	"node": {
		"nodeName": "3228k8s00000000",
		"cpu": {
			"time": "2019-04-01T04:11:13Z",
			"usageCoreNanoSeconds": 143920000000
		},
		"pods": [{
			"containers": [{
				"name": "test",
				"cpu": {
					"time": "2019-04-01T04:11:15Z",
					"usageNanoCores": 4227145,
					"usageCoreNanoSeconds": 2500000000
				}
			}],
			"cpu": {
				"time": "2019-04-01T04:10:31Z",
				"usageNanoCores": 4227145,
				"usageCoreNanoSeconds": 2500000000
			}
		}]
	}
} 
```

**What you expected to happen**:
node's usageNanoCores exists from kubelet stats summary for Windows nodes

**How to reproduce it (as minimally and precisely as possible)**:
Build k8s 1.14 cluster with a customized aks-engine tool(I did some changes to use metrics-server 0.3.1), check the metrics-sever pod logs.

**Environment**:
- Kubernetes version (use `kubectl version`):
1.14
- Cloud provider or hardware configuration:
AKS-Engine on Azure

- OS (e.g: `cat /etc/os-release`):
Windows Server 2019

/sig windows
/kind bug

@feiskyer @PatrickLang

**Comment by andyzhangx** [comment]
@zhiweiv is it related to metrics-server version? can heapster collect this metric on Windows?

**Comment by zhiweiv** [comment]
Metrics server 0.3.x uses usageNanoCores, 0.2.x uses usageCoreNanoSeconds.

usageNanoCores is missing on Windows nodes, that is why cpu metrics of Windows is empty while using 0.3.x.

**Comment by PatrickLang** [comment]
So I guess https://github.com/kubernetes/kubernetes/pull/73659 added container/pod but not node

**Comment by andyzhangx** [comment]
https://github.com/kubernetes/kubernetes/pull/80176 fixed this, need confirm

**Comment by zhiweiv** [comment]
Close this since it is fixed in 1.16.
